test/fixtures/simple_regional_with_gateway_api/example.tf

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "example" {
+  source = "../../../examples/simple_regional_with_gateway_api"
+
+  project_id                     = var.project_ids[0]
```

Comment:
Could you please update it to var.project_ids[2]? I think this would resolve the error reported by the CloudBuild.

Reply:
Hi @ericyz is there some kind of documentation regarding the differences between the project ids?
I was wondering about this before. How one decides which one to use?

Reply:
Hi @ericyz unfortunately this change didn't fix the issue. I added you as a contributor to my repository so you can more easily try to fix the issue. As I'm not able to view the error reports it's a blackbox for me and we will have to keep this back-and-forth going.